src/status_im2/common/toasts/view.cljs

```diff
@@ -0,0 +1,104 @@
+(ns status-im2.common.toasts.view
+  (:require
+   [quo2.core               :as quo2]
+   [react-native.core       :as rn]
+   [react-native.gesture    :as gesture]
+   [react-native.reanimated :as reanimated]
+   [reagent.core            :as reagent]
+   [status-im.utils.utils   :as utils.utils]
+   [utils.re-frame          :as rf]))
+
+(def ^:private slide-out-up-animation
+  (-> reanimated/slide-out-up-animation
```

Comment:
mhm interesting, but im not sure if wee need to use `^js` here for advanced mode

Reply:
fixed with oops/ocall